### PR TITLE
dx: in debug mode, make `--bun` overwrite `bun` in addition to `node`

### PR DESCRIPTION
**Pull request by paperclover**
Fixes `bun-debug repl`

### CODE REVIEW

**Comment by Jarred-Sumner** [comment]
merge conflict

**Review by Jarred-Sumner — APPROVED** [review]
LGTM after merge conflict

**Comment by paperclover** [comment]
Going to merge into my bunx branch because im modifying stuff related to this

**Pull request closed without merging**
